src/Mod/TechDraw/Gui/TaskDimension.ui

```diff
@@ -319,8 +413,6 @@ be used instead if the dimension value</string>
    <header>Gui/Widgets.h</header>
   </customwidget>
  </customwidgets>
- <resources>
-  <include location="Resources/TechDraw.qrc"/>
```

Comment:
why this deletion?

Reply:
I think I must have a bad merge in my local repo.   Too many things being marked as changes that I wasn't interested in.   I'm going to cancel this PR, check the histories and resubmit.

Thanks for your help.

Reply:
> Too many things being marked as changes that I wasn't interested in.

Please check of you have the Designer Plugin, see my last comment with the screenshot.
Without the plugin you will end up the same I assume.